game.html

```diff
@@ -30,13 +32,18 @@
             <a-assets>
 
                 <a-asset-item   id="playerModel" 
-                                src="models/low_poly_space_ship_09/scene.gltf"
->
+                                src="models/low_poly_space_ship_09/scene.gltf">
                 </a-asset-item>
                 <img    id="slideTexture" 
                         src="img/TexturesCom_Ice_Cracked_1K_albedo.png">
                 <img    id="bonusTexture" 
                         src="img/bonus.png"> 
+
+                <!-- music d-__-b -->
+                <audio id="collision" src="https://freesound.org/data/previews/156/156031_2703579-lq.mp3" preload="auto"></audio>
+                <audio id="bonus" src="https://freesound.org/data/previews/253/253172_4404552-lq.mp3" preload="auto"></audio>
+                <audio id="bonus" src="https://freesound.org/data/previews/412/412168_7383104-lq.mp3" preload="auto"></audio>
```

Comment:
You should definitely create static resources directory for audio files. Player should be able to cache these files instead of downloading them everytime he starts game.

Reply:
done